cst/plugins/eu.esdihumboldt.cst.functions.geometric/src/eu/esdihumboldt/cst/functions/geometric/NetworkExpansion.java

```diff
@@ -151,10 +151,7 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 		GeometryProperty<Geometry> result = null;
 		if (old_geometry != null) {
 			Geometry new_geometry = null;
-			BufferParameters bufferParameters = new BufferParameters();
-			bufferParameters.setEndCapStyle(CAP_STYLE);
```

Comment:
Is this the default in the new API?

Reply:
No, oversight on my part, thanks for the pointer! I'll push an update.